### DISCUSSION TITLE
Upgrade - Jump multiple release series in a standalone configuration

### DISCUSSION THREAD

**Discussion by rmjstn** [rabbitmq/rabbitmq-server#9001] [Questions] [answered]
Hello,

The documentation indicates that "[When an upgrade jumps multiple release series, it may be necessary to perform an intermediate upgrade first](https://www.rabbitmq.com/upgrade.html#rabbitmq-version-upgradability)".
I don't have a cluster but a standalone instance. Does this rule also apply to standalone mode (only one node) ?  

Thanks,

Rémy

**Comment by mkuratczyk** [comment, marked as answer]
In general - yes, although there might situations (upgrades between specific versions), where it wouldn't be necessary for a single-node. If this is a dev environment (or even production with empty queues), you can always just delete it altogether (delete data stored by RabbitMQ) and upgrade immediately to the latest version. On the other end of the spectrum, you can use https://www.rabbitmq.com/blue-green-upgrade.html to upgrade directly from even a very old version, directly to the latest (since you never cluster them together but rather you switch the apps to a completely new cluster/node).
